browser/brave_stats/BUILD.gn

```diff
@@ -0,0 +1,46 @@
+import("//brave/components/brave_referrals/buildflags/buildflags.gni")
+
+declare_args() {
+  brave_stats_api_key = ""
+  brave_stats_updater_url = "https://laptop-updates.brave.com"
```

Comment:
We're leaving these empty by default in both gn and config.js now because we don't want other browsers based on Brave to hit these endpoints. Contact @mihaiplesa about adding this to CI

Reply:
blocked by https://github.com/brave/devops/issues/3731